DfE.FindInformationAcademiesTrusts/Pages/OtherServicesLinkBuilder.cs

```diff
@@ -59,4 +62,11 @@ public string GetInformationAboutSchoolsListingLinkForAcademy(string urn)
             _ => null
         };
     }
+
+    public string? SharepointFolderLink(string groupId)
+    {
+        if (string.IsNullOrEmpty(groupId)) return null;
```

Comment:
Group ID cannot be null (thanks to nullable type constraints) and is never empty in the db. Can we remove this and make the return type of the method non-nullable for easier maintenance?